backend/sql/postgresql/schema/20231108001411_invites.up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS neosync_api.invites (
+	id uuid NOT NULL DEFAULT gen_random_uuid(),
+  account_id uuid NOT NULL,
+  sender_user_id uuid NOT NULL,
```

Comment:
I'm thinking this should be NULL and the FK constraint should set to null if the user_id is deleted. That way a user could be removed, but the invite is retained.
Any thoughts?

Reply:
yeah that makes sense